src/drivers/evdev/lv_evdev.c

```diff
@@ -171,10 +278,77 @@ lv_indev_t * lv_evdev_create(lv_indev_type_t indev_type, const char * dev_path)
 
     dsc->fd = open(dev_path, O_RDONLY | O_NOCTTY | O_CLOEXEC);
     if(dsc->fd < 0) {
-        LV_LOG_ERROR("open failed: %s", strerror(errno));
+        LV_LOG_WARN("open failed: %s", strerror(errno));
         goto err_after_malloc;
     }
 
+    struct stat sb;
+    if(0 != fstat(dsc->fd, &sb)) {
+        LV_LOG_ERROR("fstat failed: %s", strerror(errno));
+        goto err_after_open;
+    }
+    dsc->st_dev = sb.st_dev;
+    dsc->st_ino = sb.st_ino;
+
+    lv_indev_t * ex_indev = NULL;
+    while((ex_indev = lv_indev_get_next(ex_indev))) {
+        if(lv_indev_get_read_cb(ex_indev) != _evdev_read) continue;
+        lv_evdev_t * ex_dsc = lv_indev_get_driver_data(ex_indev);
+        if(dsc->st_dev == ex_dsc->st_dev && dsc->st_ino == ex_dsc->st_ino) {
```

Comment:
Not sure if it's a realistic use case: Would it be conceivable that someone might be using multiple instances of the same device, maybe with multiple displays?

Reply:
Interesting idea. I can hardly imagine it in real life, but if possible IMO we can leave this opportunity open.